Flake test test_e2e_90_kafka_events.py:: test_01_napp_sends_data_correctly

Results from last night exec:

```
   async def test_01_napp_sends_data_correctly(self):
       """
       Test that kafka_events correctly runs the 'setup' method. This would require
       that the AIOKafkaProducer has been properly initialized
       """
       # Create a consumer before the message is propagated to reduce the amount of messages

       consumer = AIOKafkaConsumer(
           (KAFKA_TOPIC),
           bootstrap_servers=KAFKA_ADDRESSES,
       )

       await consumer.start()

       # Create an EVC-creation event to send to Kafka

       evc_name = "Vlan_%s" % 902

       payload = {
           "name": evc_name,
           "enabled": True,
           "dynamic_backup_path": True,
           "uni_a": {
               "interface_id": "00:00:00:00:00:00:00:01:1",
               "tag": {"tag_type": "vlan", "value": 902}
           },
           "uni_z": {
               "interface_id": "00:00:00:00:00:00:00:03:1",
               "tag": {"tag_type": "vlan", "value": 902}
           }
       }
       api_url = KYTOS_API + '/mef_eline/v2/evc/'
       response = requests.post(
           api_url,
           data=json.dumps(payload),
           headers={'Content-type': 'application/json'},
           timeout=5
       )

       assert response.status_code == 201, response.text

       # Wait for the message to propagate

       await asyncio.sleep(4)

       # Try pulling from Kafka multiple times

       found = False

       for _ in range(4):

           if found:
               break

           try:
               # Wait up to 2 seconds for messages
               results = await consumer.getmany(timeout_ms=2000)

               # Ensure values exist
               assert results.values()

               for messages in results.values():
                   for msg in messages:
                       event = json.loads(msg.value.decode())
                       if event["event"] == 'kytos/mef_eline.created':
                           found = True
                           break

           except Exception as exc:
               print(f"An exception occurred: {exc}")

     assert found
E       assert False

```

With the retry methodology, it passed. We should investigate what happened to understand how we can get this test more stable.

--- a/tests/test_e2e_90_kafka_events.py
+++ b/tests/test_e2e_90_kafka_events.py
@@ -71,7 +71,7 @@ class TestE2EKafkaEvents:
         await self.admin.delete_topics([KAFKA_TOPIC], TIMEOUT)
 
         # Let the topic deletion propagate
-        await asyncio.sleep(2)
+        await asyncio.sleep(5)
 
     async def topic_exists(self):
         """
@@ -93,7 +93,7 @@ class TestE2EKafkaEvents:
         )
 
         # Let the topic creation propagate
-        await asyncio.sleep(2)
+        await asyncio.sleep(5)
 
     async def test_01_napp_sends_data_correctly(self):
         """

